docs/python/testing.md

```diff
@@ -12,144 +12,27 @@ MetaSocialImage: images/tutorial/python-social.png
-
-```python
-# Import the code to be tested
-import validator
-
-# Import the test framework (this is a hypothetical module)
-import test_framework
-
-# This is a generalized example, not specific to a test framework
-class Test_TestAccountValidator(test_framework.TestBaseClass):
-    def test_validator_valid_string():
-        # The exact assertion call depends on the framework as well
-        assert(validate_account_number_format("1234567890"), True)
-
-    # ...
-
-    def test_validator_blank_string():
-        # The exact assertion call depends on the framework as well
-        assert(validate_account_number_format(""), False)
-
-    # ...
-
-    def test_validator_sql_injection():
-        # The exact assertion call depends on the framework as well
-        assert(validate_account_number_format("drop database master"), False)
-
-    # ... tests for all other cases
-```
-
-The exact structure of the code depends on the test framework you're using, and specific examples are provided later in this article. In any case, as you can see, each test is simple: invoke the function with an argument and assert the expected return value.
-
-The combined results of all the tests is your test report, which tells you whether the function (the unit), is behaving as expected across all test cases. That is, when a unit passes all of its tests, you can be confident that it's functioning properly. (The practice of **test-driven development** is where you actually write the tests first, then write the code to pass increasingly more tests until all of them pass.)
-
-Because unit tests are small, isolated pieces of code (in unit testing you avoid external dependencies and use mock data or otherwise simulated inputs), they're quick and inexpensive to run. This characteristic means that you can run unit tests early and often. Developers typically run unit tests even before committing code to a repository; gated check-in systems can also run unit tests before merging a commit. Many continuous integration systems also run unit tests after every build. Running the unit test early and often means that you quickly catch **regressions,** which are unexpected changes in the behavior of code that previously passed all its unit tests. Because the test failure can easily be traced to a particular code change, it's easy to find and remedy the cause of the failure, which is undoubtedly better than discovering a problem much later in the process!
-
 For a general background on unit testing, read [Unit testing](https://wikipedia.org/wiki/Unit_testing) on Wikipedia. For useful unit test examples, you can review [https://github.com/gwtw/py-sorting](https://github.com/gwtw/py-sorting), a repository with tests for different sorting algorithms.
```

Comment:
Can we include a reference to the general testing doc for VS Code (/docs/debugtest/testing.md)?